os_net_config/sriov_config.py

```diff
@@ -574,7 +574,7 @@ def trigger_udev_rules():
         raise
 
 
-def configure_switchdev(pf_name):
+def configure_switchdev(pf_name, link_mode='legacy'):
```

Comment:
I think, lets have another function to set the ethtool hw-tc-offload and invoke it from here. The same function can be called for legacy again.